common/shareddb.cpp

```diff
@@ -51,7 +51,9 @@ namespace ItemField
 #define F(x) x,
 #include "item_fieldlist.h"
 #undef F
-		updated
+		updated,
+		minstatus,
+		comment,
```

Comment:
Extra comma

Reply:
Can leave it since it doesn't change functionality.